application/comit_node/src/http_api/rfc003/action.rs

```diff
@@ -451,7 +383,7 @@ pub fn handle_post<T: MetadataStore<SwapId>, S: StateStore<SwapId>>(
     with_swap_types_bob!(
         &metadata,
         (|| match action {
-            PostAction::Accept => serde_json::from_value::<AcceptSwapRequestHttpBody<AL, BL>>(body)
+            PostAction::Accept => serde_json::from_value::<BobAcceptBody>(body)
```

Comment:
❓  Is that a type aliasing? Is `ROLE` a type aliasing too? Should we have a format convention for type aliasing to make things easier such as all upper-case?

Reply:
That is exactly what @LLFourn suggested, which is why it was introduced here.